### PR TITLE
UNet segmentation of ISIC dataset

### PR DESCRIPTION
**Pull request by kristiannorved**
This branch creates a segmentation of the ISIC dataset using a UNet neural network architecture. The model is trained, validated, and tested on data provided by the ISIC challenge team.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
**This is an initial inspection, no action is required at this point**
- Code: unable to verify validity
- Results: unable to verify validity, blank output predicted
- Readme: images are broken in GitHub web; no description of model; no reference; no dependency
- Commit Messages: should be ore specific
- Other comments: N/A

IMPORTANT: Commits are not in the right place.  In your PR update, please fork Shakes repo in whole (there is a checkmark when you fork) and commit to his topic-recognition branch. You should be able to see work from the previous years in "recognition" folder. your work should be alongside theirs (not in root).

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Use and implementation not verifiable (minimal functionality) -5
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem - not possible to tell and no output provided, design flawed -3
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage 
Module present
Commenting
No Data leakage (no evidence of data usage, test/validation set etc.) -2
Difficulty: Easy -10
## Commit Log
Meaningful commit messages, could be more descriptive -1
Progressive commits used
## Documentation
ReadMe minimal -2
OK Description and Comments (broken links, minimal) -1
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required, but not mergeable, not functional 
Request Description minimal, more info req -1

**Comment by shakes76** [comment]
Can't identify you, can you mention your ID, username or name so that grades can be done?

**Comment by kristiannorved** [comment]
Hi there,

Sorry that’s weird that you can’t identify me. Here’s my information:

Student ID: 46406619
Name: Kristian Norved

Kind regards,

Kristian Norved
